src/development/LocalTest/Controllers/Authentication/AuthenticationController.cs

```diff
@@ -49,5 +49,52 @@ public ActionResult RefreshJWTCookie()
             logger.LogInformation($"End of refreshing token");
             return Ok(token);
         }
+
+        [HttpGet("orgToken")]
+        public ActionResult GenerateOrgToken(
+            [FromQuery] string org = "ttd",
+            [FromQuery] int orgNumber = 111111111,
+            [FromQuery] string scope = "altinn:instances.read altinn:instances.write")
+        {
+            List<Claim> claims = new List<Claim>
+            {
+                new Claim("urn:altinn:org", org, ClaimValueTypes.String),
+                new Claim("urn:altinn:orgNumber", orgNumber.ToString(), ClaimValueTypes.Integer32),
+                new Claim(AltinnCoreClaimTypes.AuthenticateMethod, "fake-local-maskinporten", ClaimValueTypes.String),
+                new Claim(AltinnCoreClaimTypes.AuthenticationLevel, "3", ClaimValueTypes.Integer32),
+                new Claim("urn:altinn:scope", scope, ClaimValueTypes.String)
+            };
+
+            ClaimsIdentity identity = new ClaimsIdentity("OrganisationLogin");
+
+            identity.AddClaims(claims);
+            ClaimsPrincipal principal = new ClaimsPrincipal(identity);
+
+            string token = jwtHandler.GenerateToken(principal, new TimeSpan(0, Convert.ToInt32(generalSettings.GetJwtCookieValidityTime), 0));
```

Comment:
In the newest version of JwtCookieHandler, the GenerateToken has implemented async, just as an fyi.